README.md

```diff
@@ -1,8 +1,15 @@
-# Operation
+Operation
+=========
+[![python-versions](https://img.shields.io/pypi/pyversions/operation)](https://img.shields.io/pypi/pyversions/operation)
+[![pypi-badge](https://img.shields.io/pypi/v/operation.svg)](https://pypi.org/project/operation)
 
 [Railway Oriented Programming](https://fsharpforfunandprofit.com/rop/) implementation for Python.
 
-### Basic Usage
+## Motivation
+Handling events that trigger a sequence of operations and side-effects can be extremely elegant using the railway pattern.
+This package can allow you to create a dedicated pipeline for each event, that will preform the tasks in an ordered fashion. 
```

Comment:
This package can allow you to create a dedicated pipeline  for each flow or event component